### DISCUSSION TITLE
Async Hooks

### DISCUSSION THREAD

**Discussion by chris-schra** [payloadcms/payload#778] [Feature Requests & Ideas]
Imagine fetching some external data to add within an afterChange hook. Right now it looks like the hook as to be synchronous, but fetch etc. will always be async.

Does it make sense to allow async hooks at least for fields?

**Comment by denolfe** [comment]
Currently, all hooks, including field hooks, support async.

**Reply by chris-schra** [reply]
made a mistake, my bad, sorry
